debug/src/debug.js

```diff
@@ -6,6 +6,23 @@ export function initDebug() {
 	/* eslint-disable no-console */
 	let oldBeforeDiff = options.diff;
 
+	options.checkParent = parentNode => {
```

Comment:
Renaming `checkParent` to `root` saves 3 bytes because the latter is already in our gzip dictionary :tada: